src/devices/bus/isa/sblaster_lle.cpp

```diff
@@ -404,9 +459,27 @@ void isa8_sblaster_2_0_lle_device::map_dsp_io(address_map &map)
 	map(0x0000, 0xffff).rw(FUNC(isa8_sblaster_2_0_lle_device::dsp_latch_r), FUNC(isa8_sblaster_2_0_lle_device::dsp_latch_w));
 }
 
+uint8_t isa8_sblaster_2_0_lle_device::saa1099_16_r(offs_t offset)
+{
+	return 0xff;
+}
+
+void isa8_sblaster_2_0_lle_device::saa1099_1_16_w(offs_t offset, uint8_t data)
+{
+	m_saa1099_1->write(offset, data);
+}
+
+void isa8_sblaster_2_0_lle_device::saa1099_2_16_w(offs_t offset, uint8_t data)
+{
+	m_saa1099_2->write(offset, data);
+}
```

Comment:
Are these trampolines needed?  Is there a reason you can’t install the device handlers directly?

Reply:
The SAA1099s are write-only and m_isa->install_device doesn't handle that case, so I still need the read handler locally but I can change the write handler to hit the device directly.